cli/cmd/generate_aws.go

```diff
@@ -0,0 +1,375 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/lwgenerate/aws"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionEnableConfig                = "Enable Config Integration?"
+	QuestionEnableCloudtrail            = "Enable Cloudtrail Integration?"
+	QuestionAwsRegion                   = "Specify the AWS region Cloudtrail, SNS, and S3 resources should use:"
```

Comment:
Nit: Personally I prefer this wording.
`Specify the AWS region to be used by Cloudtrail, SNS, and S3:`
(non blocking)

Reply:
👍